test/distributed/test_c10d.py

```diff
@@ -2223,6 +2223,17 @@ def __init__(self):
     def forward(self, x):
         return F.softmax(self.embedding(x), dim=1)
 
+class MixedDenseSparseGradientModule(nn.Module):
+    def __init__(self):
+        super(MixedDenseSparseGradientModule, self).__init__()
```

Comment:
nit: this is prior to this PR. We are on Python 3.6+ now, this can be simplified to `super().__init__()`. Please feel free to edit existing tests to use PY3 conventions if you like.

Reply:
I updated my Module.